src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -120,8 +119,21 @@ public async override System.Threading.Tasks.Task RunTaskAsync ()
 						}
 					}
 				}
-				if (!string.IsNullOrEmpty (ProguardRuleOutput))
-					Files.CopyIfChanged (proguardRuleOutputTemp, ProguardRuleOutput);
+				if (!string.IsNullOrEmpty (ProguardRuleOutput)) {
+					// combine the "proguard" temp files into one file.
+					var sb = new StringBuilder ();
+					HashSet<string> output = new HashSet<string> ();
+					output.Add ("#Auto Generated file. Do not Edit.");
```

Comment:
I think this should actually be `sb.AppendLine ("#Auto Generated …")`.  By adding the string to `output`, you don't know where it will wind up in the output; it could be at the end of the file, or the middle…